rebase_automation.sh

```
@@ -0,0 +1,527 @@
+            
+            # Update skipRange
+            sed -i "s/olm.skipRange: '>=.*<.*'/olm.skipRange: '>=$OLD_BUNDLE_VERSION <$NEW_BUNDLE_VERSION'/" "$csv_file"
+            
+            # Note: Description updates will be handled in Step 4 (manual replacements)
+        fi
+    done
+    
+    # Update bundle.Dockerfile
+    if [[ -f "bundle.Dockerfile" ]]; then
+        log_info "Updating bundle.Dockerfile"
+        local old_channel_version=$(echo "$OLD_BUNDLE_VERSION" | cut -d'.' -f1,2)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)